addon/types/model_export/model.py

```diff
@@ -362,7 +363,7 @@ def view_model(self):
         # Use wine to run HLMV on Linux.
         # Wine tends to complain about the paths we feed HLMV.
         # So we use relatve paths working from the base directory of the game.
-        if (os.name == 'posix') and (self.studiomdl.suffix == '.exe'):
+        if (os.name == 'posix') and (self.hlmv.suffix == '.exe'):
```

Comment:
Is there a native binary for HLMV as well?

Reply:
Unfortunately no, since it heavily relies on win32 api junk

Reply:
Ah alright, then I suppose there's no need to support a hypothetical native version.

Reply:
For Strata at least, we plan on having a native model browser but it won't be called hlmv. I will add support for that here when we ship it